README.rst

```diff
@@ -72,6 +72,7 @@ Installing
 ----------
 
 ``autoswitch-virtualenv`` requires `virtualenv <https://pypi.org/project/virtualenv/>`__ to be installed.
+You'll also need to make sure that `python` (without a suffix; both Python 2 and 3 are supported) is available in your `$PATH`.
```

Comment:
I could add a note here that recommends installing `python-is-python3` on Ubuntu-based distros... But one could also consider that out-of-scope for this document :) If it is not added here, I'll at least document it in the original ticket (#171)